draft-ietf-rtcweb-jsep.xml

```diff
@@ -1488,7 +1483,7 @@ Suggested:
           gathered. As "true" is the only value that definitively
           indicates remote Trickle ICE support, an application that
           compares canTrickleIceCandidates against "true" will by
-          default attempt Half Trickle on initial offers and Full
+          default attempt Half Trickle on initial offers (as canTrickleIceCandidates will be null) and Full
           Trickle on subsequent interactions with a Trickle
```

Comment:
```suggestion
          Trickle on subsequent interactions, when negotiating with a Trickle
```

Reply:
Added this tweak to make it clear that we're talking about a trickle-compatible agent for both the initial and subsequent interactions. LMK if you agree.

Reply:
Not sure it's necessary but it seems fine.

Reply:
Rereading this change, I think this makes the sentence a bit clumsier and harder to understand, so at this point in time I think we should just keep the original text.